src/opm/output/eclipse/Summary.cpp

```diff
@@ -134,105 +136,86 @@ namespace {
         };
     }
 
-    std::vector<Opm::SummaryConfigNode>
+    std::vector<Opm::EclIO::SummaryNode>
     requiredRestartVectors(const ::Opm::Schedule& sched)
     {
-        auto entities = std::vector<Opm::SummaryConfigNode>{};
-
-        using SN = ::Opm::SummaryConfigNode;
-
         const auto& vectors = requiredRestartVectors();
+        static const std::vector<ParamCTorArgs> extra_well_vectors {
+            { "WBHP",  Opm::EclIO::SummaryNode::Type::Pressure },
+            { "WGVIR", Opm::EclIO::SummaryNode::Type::Rate     },
+            { "WWVIR", Opm::EclIO::SummaryNode::Type::Rate     },
+        };
+        static const std::vector<ParamCTorArgs> extra_group_vectors {
+            { "GMCTG", Opm::EclIO::SummaryNode::Type::Mode },
+            { "GMCTP", Opm::EclIO::SummaryNode::Type::Mode },
+            { "GMCTW", Opm::EclIO::SummaryNode::Type::Mode },
+        };
+        static const std::vector<ParamCTorArgs> extra_field_vectors {
+            { "FMCTG", Opm::EclIO::SummaryNode::Type::Mode },
+            { "FMCTP", Opm::EclIO::SummaryNode::Type::Mode },
+            { "FMCTW", Opm::EclIO::SummaryNode::Type::Mode },
+        };
```

Comment:
Please don't mark these objects as `static`.  The `requiredRestartVectors()` function is called exactly once&mdash;as part of the single-threaded [SMSPEC configuration](https://github.com/OPM/opm-common/blob/d40ebe647a9b808c115daa05e52535004b732919/src/opm/output/eclipse/Summary.cpp#L2478)&mdash;and there's no need to keep the objects in memory for the duration of the process just for that configuration.